works/templates/works/history_new_hx.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'hx_modal_form.html' %}
```

Comment:
I don't think this file has the correct name i think it should be called something like: item_state_change_hx

Reply:
it's consistent with the existing file :)

Reply:
Ahh yes i never noticed that that was called history_new.html now they both feel silly